src/taipy/gui/gui_actions.py

```diff
@@ -111,19 +111,38 @@ def resume_control(state: State):
         warnings.warn("'resume_actions()' must be called in the context of a callback")
 
 
-def navigate(state: State, to: t.Optional[str] = ""):
+def navigate(state: State, to: t.Optional[str] = "", tab: t.Optional[str] = None):
     """Navigate to a page.
 
     Arguments:
         state (State^): The current user state as received in any callback.
-        to: The name of the page to navigate to. This must be a valid page identifier.
+        to: The name of the page to navigate to. This can be a page identifier.
```

Comment:
I guess this is pretty unclear.
"This can be a page identifier" ... or what?
What about the leading '/'?

Reply:
"to: The name of the page to navigate to. This can be a page identifier (as created by `Gui.add_page^` with no leading /) or an URL."

Reply:
The "or an URL" clarifies the thing I guess